tests/transforms/test_optimization/utils.py

```diff
@@ -31,29 +48,42 @@ def compute_matrix_from_ops_one_qubit(ops):
 def compute_matrix_from_ops_two_qubit(ops, wire_order):
     """Given a list of two-qubit operations, construct its matrix representation."""
 
-    mat = qml.math.kron(I, I)
+    mat = qml.math.eye(4)
+
+    wire_order = qml.wires.Wires(wire_order)
 
     for op in ops:
-        if len(op.wires) == 1:
-            if op.wires == qml.wires.Wires(wire_order[0]):
-                mat = qml.math.dot(qml.math.kron(op.matrix, I), mat)
+        op_wires = qml.wires.Wires(op.wires)
+
+        if op.name == "RZ" and qml.math.get_interface(op.parameters[0]) == "tensorflow":
```

Comment:
I added this to fix the tests. Since we will soon have #1609 available, this function will be obsolete anyways.